docs/backlog-management.md

```diff
@@ -0,0 +1,79 @@
+# Issue Management for OpenTelemetry
+
+It's an important community goal for OpenTelemetry that our members find the backlogs to be responsive, and easy to take part in. Having a shared methodology for processing issues helps new members understand the current state of each project.
+
+SIGs are encouraged to experiment with labels and backlog management procedures. This document only covers the bare bones of issue management which should work the same across all SIGs, to help maintain a responsive backlog and allow us to track work across all projects in a similar manner.
+
+## New Issues
+
+Flow:
+* An issue is filed by a community member.
+* A maintainer responds and asks clarifying questions.
+* The maintainer processes the issue as a Code Change, a Discussion, or resolves it quickly.
+
+Every new issue should be responded to quickly by a Maintainer, with a greeting and any needed clarifying questions. If assessing the issue requires more information, the Triager collaborates with the OP until workability is determined. If the ticket is a help request with an existing answer, please link to the answer before closing the ticket.
+
+## Code Changes
+
+Flow:
+* A new issue is filed, proposing a code change.
+* Maintainer labels the issue as `up-for-grabs`; reviewers are assigned.
+* `up-for-grabs` becomes `wip` once a Collaborator is assigned.
+* Closed once all PRs are accepted.
+
+A ticket becomes `up-for-grabs` once a maintainer deems it is processed sufficiently that work can begin. Issues should be well scoped and have clear goals.
+
+A ticket labeled `up-for-grabs` can be picked up by anyone in the community. Available work can be assigned in the following ways:
+* A contributor assigns the ticket to themselves.
+* A maintainer assigns the ticket to a contributor.
+
+All tickets transition to `wip` once they have been assigned. If a PR is merged which resolves the issue, the issue can be closed. If work is abandoned, or the contributors otherwise change their minds, the ticket should be closed as `no-further-action`.
+
+## Discussions
```

Comment:
please cover the case of `spec-needed` label like here: https://github.com/open-telemetry/opentelemetry-node/issues/5